src/docs/api/characteristics.md

```diff
@@ -37,6 +37,8 @@ switchService.getCharacteristic(this.api.hap.Characteristic.On)
   });
 ```
 
+Implementation of get event handler is not the recommended method of updating as accessories status and the use of [service.updateCharacteristic](https://github.com/homebridge/homebridge.github.io/blob/source/src/docs/api/service.md#serviceupdatecharacteristic) is recommended to provide a robust user experience.
```

Comment:
We might want to specify the difference to this method, like possible restrictions of not having the possibility of doing on the fly calculations/executions. Though also highlighting that we don't want such behavior, where expensive operations are kicked off for every single read.

Reply:
Added more commentary